poetry.lock

```diff
@@ -733,6 +748,17 @@ category = "dev"
 optional = false
 python-versions = ">=3.7"
 
+[[package]]
+name = "multiprocess"
+version = "0.70.16"
+description = "better multiprocessing and multithreading in Python"
+category = "dev"
+optional = false
+python-versions = ">=3.8"
+
+[package.dependencies]
+dill = ">=0.3.8"
+
 [[package]]
 name = "mypy"
 version = "0.991"
```

Comment:
```suggestion
version = "1.1"
```

Reply:
Will update this manually via `poetry lock` and commit the changes